bundles/org.openhab.core.addon/src/main/java/org/openhab/core/addon/internal/xml/AddonDiscoveryMethodConverter.java

```diff
@@ -22,6 +22,7 @@
 import org.openhab.core.config.core.xml.util.GenericUnmarshaller;
 import org.openhab.core.config.core.xml.util.NodeIterator;
 
+import com.thoughtworks.xstream.converters.Converter;
```

Comment:
Why do you use an import here instead of using the fully qualified class name  (like in the other classes)? IIRC IntelliJ complains about "unused imports" if you import classes that are only used in JavaDoc

Reply:
I found it was fixed similarly in other classe like `BridgeTypeConverter` and `ChannelConverter`.
Do you see the warning there as well?
I don't see this in my IntelliJ, but maybe we use a different configuration. :slightly_smiling_face:

Reply:
Looks good. I'm pretty sure there was an issue with that, either spotless removed the import (because it was "unused") or IntelliJ complained. In the first case it could be that this was solved by one of the latest spotless upgrades. Or it may depend on if other classes from the same package are imported, too.